dokka-runners/dokka-gradle-plugin/src/testFunctional/kotlin/MigrationMessagesTest.kt

```diff
@@ -26,141 +28,203 @@ class MigrationMessagesTest : FunSpec({
-                |│       org.jetbrains.dokka.experimental.gradle.pluginMode=V2EnabledWithHelpers        │
-                |│   to your project's `gradle.properties`                                              │
-                |│                                                                                      │
-                |│   Please report any feedback or problems to Dokka GitHub Issues                      │
-                |│       https://github.com/Kotlin/dokka/issues/                                        │
-                |└──────────────────────────────────────────────────────────────────────────────────────┘
-                """.trimMargin()
-            output shouldNotContain "Dokka Gradle Plugin V2"
-        }
+        private val expectedV1Warning = /* language=text */ """
+            |warning: Dokka Gradle plugin V1 is deprecated
+            """.trimMargin()
 
-        private fun BuildResult.shouldOnlyContainV2Message() {
-            output shouldContainOnlyOnce /* language=text */ """
-                |┌──────────────────────────────────────────────────────────────────────┐
-                |│ Dokka Gradle Plugin V2 is enabled ♡                                  │
-                |│                                                                      │
-                |│   We would appreciate your feedback!                                 │
-                |│   Please report any feedback or problems to Dokka GitHub Issues      │
-                |│       https://github.com/Kotlin/dokka/issues/                        │
-                |│                                                                      │
-                |│   If you need help or advice, check out the migration guide          │
-                |│       https://kotl.in/dokka-gradle-migration                         │
-                |│                                                                      │
-                |│   You can suppress this message by adding                            │
-                |│       org.jetbrains.dokka.experimental.gradle.pluginMode.noWarn=true │
-                |│   to your project's `gradle.properties`                              │
-                |└──────────────────────────────────────────────────────────────────────┘
-                """.trimMargin()
-            output shouldNotContain "Dokka Gradle Plugin V1"
-        }
+        private val expectedV1Message = /* language=text */ """
+            |Dokka Gradle plugin V1 is deprecated, and will be removed in Dokka version 2.1.0
+            |Please migrate to Dokka Gradle plugin V2. This will require updating your project.
+            |To learn about migrating read the migration guide https://kotl.in/dokka-gradle-migration
+            |
+            |To start migrating to Dokka Gradle plugin V2 add
+            |    ${PLUGIN_MODE_FLAG}=V2EnabledWithHelpers
```

Comment:
minor: I think for tests it's better to inline the string, so that we really see what the warning is and check if it's for some reason changed

Reply:
I would agree, if `PLUGIN_MODE_FLAG` was the same as `PluginFeaturesService#PLUGIN_MODE_FLAG`, but the test redefines the flags. So if the flags change in `PluginFeaturesService`, then the test will fail.